File: Latihan2-cpp/Product.cpp
#include <iostream>
#include <string>

using namespace std;

class Product{

    private:

    string price;
    string idProduct;

    public:

    // Constructor
    Product(){
    }

    Product(string price, string idProduct){
        this->price = price;
        this->idProduct = idProduct;
    }

    // Get-set Price
    void setPrice(string price){
        this->price = price;
    }

    string getPrice(){
        return this->price;
    }

    // Get-set ID Product
    void setIdProduct(string idProduct){
        this->idProduct = idProduct;
    }

    string getIdProduct(){
        return this->idProduct;
    }

    // Deconstructor
    ~Product(){
    }

};

File: Latihan2-py/Product.py
class Product:
    # Constructor
    def __init__(self):
        self.__price = ""
        self.__idProduct = ""
    
    # Get-set Price
    def setPrice(self, price):
        self.__price = price

    def getPrice(self):
        return self.__price
    
    # Get-set ID Product
    def setIdProduct(self, idProduct):
        self.__idProduct = idProduct

    def getIdProduct(self):
        return self.__idProduct